testsuite/fixtures/crates/libfoo_git/config

```diff
@@ -0,0 +1,6 @@
+[core]
+	repositoryformatversion = 0
+	filemode = true
+	bare = true
+[remote "origin"]
+	url = /home/jano/prog/alire/testsuite/fixtures/crates/libfoo/
```

Comment:
How is this setup specific path will be used?

Reply:
Uuups. I did a clone of a test repo so that origin got stored but should not have been checked in. The libfoo_git is itself a bare repository that is cloned, so its origins aren't really used. I will clean this up.